src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx

```diff
@@ -40,14 +47,18 @@ export default class ExpressionEnrichmentContainer extends React.Component<IExpr
     @observable.ref highlightedRow:ExpressionEnrichmentRow|undefined;
 
     @computed get data(): ExpressionEnrichmentRow[] {
-        return getExpressionRowData(this.props.data, this.props.store.hugoGeneSymbols);
+        return getExpressionRowData(this.props.data, this.props.store ? this.props.store.hugoGeneSymbols : []);
     }
 
     @computed get excludeGenes():string[]|null {
-        if (this.props.store.selectedMolecularProfiles.isComplete &&
-            this.props.store.selectedMolecularProfiles.result
-                .findIndex(x=>x.molecularProfileId === this.props.selectedProfile.molecularProfileId) > -1) {
-            return this.props.store.hugoGeneSymbols;
+        if (this.props.store) {
+            if (this.props.store.selectedMolecularProfiles.isComplete &&
```

Comment:
may be combine both if conditions

Reply:
good catch. ill do the same in AlterationEnrichmentContainer